src/options.cpp

```diff
@@ -1104,10 +1104,29 @@ void options_manager::init()
 
     mOptionsSort["debug"]++;
 
-    OPTIONS["INITIAL_POINTS"] = cOpt("debug", _("Initial points"),
-                                     _("Initial points available on character generation."),
-                                     0, 1000, 6
-                                    );
+    optionNames["freeform"] = _("Freeform");
+    optionNames["onepool"] = _("One pool");
+    optionNames["multipool"] = _("Separate pools");
+
+    OPTIONS["POINT_DISTRIBUTION"] = cOpt( "debug", _("Point distribution"),
```

Comment:
Especially since it's only enforced in character creation, wouldn't it be better to have pool selection be part of the character creation workflow instead of an option?